pkg/velero/backupstoragelocationlist.go

```diff
@@ -0,0 +1,63 @@
+package velero
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/golang/glog"
+	"github.com/openshift-kni/eco-goinfra/pkg/clients"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// ListBackupStorageLocationBuilder returns backupstoragelocation inventory in the given namespace.
```

Comment:
Can we have it in the same file with backupstoragelocation?

Reply:
We put list functions into a separate file for organization purposes